src/app/modules/it-systems/it-system-usages/it-system-usage-details/it-system-usage-details-relations/modify-relation-dialog/modify-relation-dialog.component.html

```diff
@@ -15,8 +16,7 @@
         formName="interface"
         [includeItemDescription]="true"
         itemDescriptionField="interfaceId"
-        descriptionLabelTitle="Snitflade ID:"
-        i18n-descriptionLabelTitle
+        [descriptionLabelTitle]="'Snitflade ID:'"
```

Comment:
I think this line can be reverted to the old version so it's more readable
```suggestion
        descriptionLabelTitle="Snitflade ID:"
```